src/webui/www/private/scripts/client.js

```diff
@@ -24,8 +24,8 @@
 
 // TODO: 'use strict';
```

Comment:
Why the TODO?

Reply:
Some upstream code in private\scripts\lib does not comply to strict mode.

Reply:
I admittedly don't understand why this is a blocker for us. Do the upstream functions require some code that won't comply to strict mode?

Reply:
It seems code in client.js calls some upstream functions which are not compatible with strict mode.

Reply:
After fixing mocha-init.js, this can be enabled too!